gcalbot/gcalbot/accounts.go

```diff
@@ -60,7 +63,12 @@ func (h *Handler) accountsListHandler(msg chat1.MsgSummary) error {
 	}
 
 	accountListMessage := "Here are your connected accounts:" + strings.Repeat("\n• %s", len(accounts))
-	_, err = h.kbc.SendMessageByConvID(msg.ConvID, accountListMessage, accounts...)
+	accountInterfaces := make([]interface{}, len(accounts))
```

Comment:
no need for this -- can just pass `accounts` in directly similar to `fmt.Sprintf` https://golang.org/pkg/fmt/#Sprintf

Reply:
Go doesn't seem to be a fan of that:
```
gcalbot/accounts.go:71:40: cannot use accounts (type []string) as type []interface {} in argument to h.kbc.SendMessageByConvID
```
https://stackoverflow.com/a/12754757